src/ui/components/Events/Event.tsx

```diff
@@ -31,15 +162,142 @@ export const getEventLabel = (event: ReplayEvent) => {
   return label;
 };
 
+const FIND_MOUSE_EVENT_TARGET_AND_REACT_CLICK_HANDLER = `
+  // Outer body runs in scope of the "current" click handler.
+  // Grab the click handler's arguments.
+  const args = [...arguments];
+  
+  (function findEventTargetAndClickHandler() {
+    // One of the args should be a mouse event
+    const foundMouseEvent  = args.find(a => typeof a === "object" && a instanceof MouseEvent);
+
+    if (foundMouseEvent) {
+      // We _could_ probably just return the onClick function or undefined here
```

Comment:
Not sure I understand this comment. Are you saying we could get rid of the wrapper `{ target: ... }` object?

Reply:
Yeah. I originally was returning an object with 3-5 fields here for debugging purposes (target node, possible `onClick` prop, possible ancestor node, `string[]` serialized ancestor tree, etc).  In the end the only field the outer logic actually ends up using is the potential `onClick` prop, since we're getting the vanilla JS function location from the execution point stack frame.